Cargo.toml

```
@@ -146,19 +149,19 @@ sha1 = "0.10.5"
 sha2 = "0.10.6"
 similar-asserts = "1.4.2"
 smallvec = { version = "1.11.2", features = ["serde"] }
-sqlparser = { git = "https://github.com/getsentry/sqlparser-rs", rev = "1bfe51c3db3e5b6bda4f52c56860af378d8ec2d3" }
+sqlparser = { version = "0.44.0" }
```

maybe 
```suggestion
sqlparser = "0.44.0"
```

Since there are no features are enabled.